src/_http.js

```diff
@@ -84,11 +84,12 @@ HttpClient.prototype.execute = function(method, path, body, query, options) {
 
   var headers = this._headers
   headers['Authorization'] = secret && secretHeader(secret)
-  headers['X-FaunaDB-API-Version'] = APIVersion
-  headers['X-Fauna-Driver'] = 'Javascript'
   headers['X-Last-Seen-Txn'] = this._lastSeen
   headers['X-Query-Timeout'] = queryTimeout
 
+  console.debug('request headers for test ')
```

Comment:
Was it added just for tests, or you want to keep them intentionally?

Reply:
done